Faker adds path objects to sys.path_importer_cache, breaking other packages

* Faker version: 6.6.3
* OS: Gentoo Linux

After importing `faker`, entries with `PosixPath` objects are added as keys to `sys.path_importer_cache`. However, the keys are supposed to be regular `str`s there, and the wrong type breaks software relying on `str` methods being available, e.g. astroid:

```
___________________________________________ ClassNodeTest.test_slots_added_dynamically_still_inferred ____________________________________________

self = <tests.unittest_scoped_nodes.ClassNodeTest testMethod=test_slots_added_dynamically_still_inferred>

    def tearDown(self):
        del sys.path[0]
        datadir = find("")
        for key in list(sys.path_importer_cache):
>           if key.startswith(datadir):
E           AttributeError: 'PosixPath' object has no attribute 'startswith'

tests/resources.py:41: AttributeError
```

Note that since Faker installs a pytest plugin, it is autoloaded by default in all programs' test suites.

### Steps to reproduce

```
import sys
import faker
print(sys.path_importer_cache)
```

### Expected behavior

The printed dict should only contain `str` keys.

### Actual behavior

```
[...] PosixPath('/usr/lib/python3.9/site-packages/faker/providers/address'): FileFinder(PosixPath('/usr/lib/python3.9/site-packages/faker/providers/address')), PosixPath('/usr/lib/python3.9/site-packages/faker/providers/automotive'): FileFinder(PosixPath('/usr/lib/python3.9/site-packages/faker/providers/automotive')), [...]
```

--- a/faker/utils/loading.py
+++ b/faker/utils/loading.py
@@ -22,7 +22,7 @@ def get_path(module: ModuleType) -> str:
     else:
         # unfrozen
         path = Path(module.__file__).parent
-    return path
+    return str(path)
 
 
 def list_module(module: ModuleType) -> List[str]:

